static/js/popovers.js

```diff
@@ -1177,89 +1074,8 @@ export function register_click_handlers() {
-        const message_id = $(e.currentTarget).data("message-id");
-        const $row = message_lists.current.get_row(message_id);
-        const message = message_lists.current.get(rows.id($row));
-
-        hide_actions_popover();
-
-        if ($row) {
-            if (message.collapsed) {
-                condense.uncollapse($row);
-            } else {
-                condense.collapse($row);
-            }
-        }
-
-        e.stopPropagation();
-        e.preventDefault();
-    });
-    $("body").on("click", ".popover_edit_message, .popover_view_source", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        const $row = message_lists.current.get_row(message_id);
-        hide_actions_popover();
-        message_edit.start($row);
-        e.stopPropagation();
-        e.preventDefault();
-    });
-    $("body").on("click", ".popover_move_message", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        const message = message_lists.current.get(message_id);
-        hide_actions_popover();
-        stream_popover.build_move_topic_to_stream_popover(
-            message.stream_id,
-            message.topic,
-            message,
-        );
-        e.stopPropagation();
-        e.preventDefault();
-    });
-    $("body").on("click", ".rehide_muted_user_message", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        const $row = message_lists.current.get_row(message_id);
-        const message = message_lists.current.get(rows.id($row));
-        const message_container = message_lists.current.view.message_containers.get(message.id);
-
-        hide_actions_popover();
-
-        if ($row && !message_container.is_hidden) {
-            message_lists.current.view.hide_revealed_message(message_id);
-        }
-
-        e.stopPropagation();
-        e.preventDefault();
-    });
-    $("body").on("click", ".view_edit_history", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        const $row = message_lists.current.get_row(message_id);
-        const message = message_lists.current.get(rows.id($row));
-
-        hide_actions_popover();
-        message_edit_history.show_history(message);
-        $("#message-history-cancel").trigger("focus");
-        e.stopPropagation();
-        e.preventDefault();
-    });
-
-    $("body").on("click", ".view_read_receipts", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        hide_actions_popover();
-        read_receipts.show_user_list(message_id);
-        e.stopPropagation();
-        e.preventDefault();
-    });
-
-    $("body").on("click", ".delete_message", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        hide_actions_popover();
-        message_edit.delete_message(message_id);
-        e.stopPropagation();
-        e.preventDefault();
-    });
 
     clipboard_enable(".copy_link").on("success", (e) => {
```

Comment:
Hmm, should this function be moved into `popover_menus.js`? Unless I'm mistaken, it's a click handler that's just for this popover.